src/AS_02_ACES.cpp

```diff
@@ -268,6 +274,12 @@ AS_02::ACES::create_4122_type5_id(const byte_t* subject_name, Kumu::fsize_t size
   // Derive the asset ID from the digest. Make it a type-5 UUID
   byte_t buf[Kumu::UUID_Length];
   memcpy(buf, bin_buf, Kumu::UUID_Length);
+#else
+  byte_t buf[Kumu::UUID_Length];
+  Kumu::GenRandomUUID(buf);
```

Comment:
This modification makes asdcplib compile without OpenSSL, but the resulting track files would be non-compliant:
Calculation of the SHA-1 hash for Ancillary Resources is required per SMPTE ST 2067-50 and cannot be replaced by generating a random UUID.

Reply:
Good catch. 
Would it be acceptable to only include the ACES implementation when OpenSSL was defined?

Reply:
> Would it be acceptable to only include the ACES implementation when OpenSSL was defined?

Definitely not ideal.

What about defining/importing a SHA-1 function if OpenSSL is not used?

Reply:
I agree with @palemieux. A SHA-1 implementation would also allow to keep AS-02 TimedText wrapping as a feature.